src/index.ts

```diff
@@ -37,7 +37,10 @@ module.exports = {
     'ja-no-abusage': true,
     'ja-no-redundant-expression': true,
     'no-mixed-zenkaku-and-hankaku-alphabet': true,
-    'ja-keishikimeishi': true,
+    'ja-keishikimeishi': {
+      "detection_hou_kata" : false,
+      "detection_ue" : false
```

Comment:
こまいですが、シングルクォーテーションが良いかと！

```suggestion
      'detection_hou_kata' : false,
      'detection_ue' : false
```